opdracht1.js

```diff
@@ -0,0 +1,38 @@
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+let allTvNames = [];
+inventory.map(function (value){
+    allTvNames.push(value.type);
+});
+
+console.log(allTvNames);
+
+// Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+let soldOutTvs = [];
+inventory.map(function (value){
+    if (value.originalStock === value.sold){
+        soldOutTvs.push(value);
+    }
+});
+console.log(soldOutTvs);
+
+// Opdracht 1c: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight beschikken. Log de uitkomst in de console.
+let haveAmbiLight = [];
+inventory.find(function (value ){
+    if (value.options.ambiLight == true){
+        haveAmbiLight.push(value);
+    }
+});
```

Comment:
Een .find vind maar één object die aan het predicaat voldoet. De manier zoals je 'm hier gebruikt is als een forEach. Als je alle tv's met ambiLight wilt hebben dan zul je wederom een .filter moeten/kunnen gebruiken.